runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsPlugin.java

```diff
@@ -45,6 +52,18 @@ public class TileIndicatorsPlugin extends Plugin
 	@Inject
 	private TileIndicatorsOverlay overlay;
 
+	@Inject
+	private Client client;
+
+	@Getter
+	private WorldPoint lastPlayerPosition = null;
```

Comment:
Object fields default to `null` values anyway, so this doesn't need to be explicitly defined.
```suggestion
	private WorldPoint lastPlayerPosition;
```